fe/src/pages/ProductPage/ProductPage.tsx

```diff
@@ -0,0 +1,64 @@
+import {
+  Flex,
+  Box,
+  Input,
+  Text,
+  FormLabel,
+  Textarea,
+  Button,
+} from '@chakra-ui/react';
+import {
+  Formik,
+  Form,
+} from 'formik';
+
+interface MyFormValues {
+  name: string;
+  description: string;
+}
+
+function ProductPage(): JSX.Element {
+  const initialValues: MyFormValues = { name: '', description: '' };
+  return (
+    <Flex direction="column" align="center">
+      <Text fontSize="30px" mb="30px">Product</Text>
+      <Box align="center">
+        <Formik
+          initialValues={initialValues}
+          onSubmit={values => {
+            console.log({ values });
+          }}
+        >
+          {({
+            values, handleChange, handleBlur, handleSubmit,
+          }) => (
+            <Form onSubmit={handleSubmit}>
+              <FormLabel>Item name</FormLabel>
+              <Input
+                mb="30px"
```

Comment:
Margin should not be given in pixels, but in some relative value. Try using `rem` value here and on other margin places instead.